test/mocha/cli.js

```diff
@@ -100,4 +100,14 @@ describe("bin/uglifyjs", function () {
            done();
        });
     });
+    it("Should work with --define", function (done) {
+       var command = uglifyjscmd + ' test/input/issue-1467/sample.js --define D=5 -cm';
```

Comment:
I think `-cm` could be replaced with `-c`. I only put mangle in the post out of habit.